kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListItemView.html

```diff
@@ -17,8 +17,8 @@
 <div data-field="view" class="list-group-item">
     <div class="list-view-pf-actions">
         <button data-type-field="edit-button" class="btn btn-sm btn-default" data-i18n-key="Edit" data-toggle="tooltip" data-delay="500" data-placement="bottom"></button>
-        <button data-type-field="save-button" class="btn btn-sm btn-default" data-i18n-key="Save" data-toggle="tooltip" data-delay="500" data-placement="bottom"></button>
-        <button data-type-field="close-button" class="btn btn-sm btn-default close-button" data-toggle="tooltip" data-delay="500" data-placement="bottom">✖</button>
+        <button data-type-field="save-button" class="btn btn-sm btn-default btn-primary" data-i18n-key="Save" data-toggle="tooltip" data-delay="500" data-placement="bottom"></button>
```

Comment:
I think we could clean this element by removing the `btn-default` class:
```suggestion
        <button data-type-field="save-button" class="btn btn-sm btn-primary" data-i18n-key="Save" data-toggle="tooltip" data-delay="500" data-placement="bottom"></button>
```